requirements/prod.in

```diff
@@ -45,15 +45,15 @@ psycopg[binary]==3.1.19
 PyGithub==2.6.1
 pyOpenSSL==25.0.0
 PyYAML==6.0.2
-qrcode==8.0
+qrcode==8.1
 querystringsafe-base64==1.1.1  # Pinned to maintain stub attribution signatures https://github.com/mozilla/bedrock/issues/11156
 requests>=2.32.3  # min secure version
 rich-text-renderer==0.2.8
 sentry-processor==0.0.1
-sentry-sdk==2.24.1
+sentry-sdk==2.26.1
 supervisor==4.2.5
 timeago==1.0.16
-https://github.com/mozmeao/wagtaildraftsharing/archive/refs/tags/mozmeao-0.2.0.tar.gz#egg=wagtaildraftsharing
```

Comment:
is it intentional that we're no longer using our fork?

Reply:
yep, see #16203